net/haproxy/src/etc/inc/plugins.inc.d/haproxy.inc

```diff
@@ -37,8 +37,7 @@ function haproxy_syslog()
     $syslogconf = array();
 
     $syslogconf['haproxy'] = array(
-        'local' => '/var/haproxy/var/run/log',
-        'facility' => array('haproxy'),
+        'facility' => array('haproxy')
```

Comment:
```suggestion
        'facility' => array('haproxy'),
```

trailing commas are nice, they reduce diff size

Reply:
sorry for that)